frontend/admin/resources/js/types/react-table-config.d.ts

```diff
@@ -121,7 +121,7 @@ declare module "react-table" {
 
   export interface Cell<
     D extends Record<string, unknown> = Record<string, unknown>,
-    V = any,
+    V = unknown,
```

Comment:
Not sure why this is here, but related to react-table. To ignore

```suggestion
    // eslint-disable-next-line @typescript-eslint/no-unused-vars
    V = unknown,
```
```suggestion
    V = unknown,
```